roles/os-loganalyze/tasks/main.yml

```diff
@@ -0,0 +1,42 @@
+---
+- name: install required packages
+  apt:
+    name: "{{ item }}"
+  with_items:
+    - build-essential
+    - libffi-dev
+    - libssl-dev
+    - python-dev
+    - git
+
+- name: create dirs
+  file:
+    state: directory
+    path: "{{ item }}"
+    mode: 0755
+  with_items:
+    - /etc/os_loganalyze
+    - /etc/os_loganalyze/wsgi
```

Comment:
You could just list the wsgi child path. All leading parent paths will be created with the provided details.